inc/settings.php

```diff
@@ -125,7 +125,7 @@ function siteorigin_north_settings_init(){
 				'sidebar_width' => array(
 					'label'             => __( 'Sidebar Width', 'siteorigin-north' ),
 					'type'              => 'measurement',
-					'live'              => true,
+					'live'              => false,
```

Comment:
@gregpriday how do you suggest we add min & max values. The setting allows for a user to choose a number of different units, but i'm guessing most users will stick to px, % or em.

An alternate approach to the issue could be using CSS. we could have something like `min-width: 100px` and `overflow-x: hidden` applied to the sidebar. In addition we could add a description mentioning that a min width has been set. Thoughts?